.env.vagrant

```diff
@@ -17,6 +17,7 @@ SUPPORTED_MIME_TYPES='application/pdf,image/jpeg,image/png,application/vnd.openx
 MAX_NUMBER_OF_FILES=10
 
 FILE_TRANSFER_FEATURE=1
+ILLNESS_REASON_FEATURE=1
```

Comment:
We are bing bit inconsistent in making toggles.

We have two formats:
1. `COMPANY_AUTH_VERIFICATION_FEATURE_ENABLED`
2. `ILLNESS_REASON_FEATURE`

I prefer the first one due to being more descriptive but we should strive to keep one format only. We should also remove file upload toggle.

Reply:
Yes, i agree with having a consistent format across the app. I've updated the feature toggles to the 1st format. I'll remove the file upload toggle in another PR